### PR TITLE
Update deploy-apps-microsoft-teams-landing-page.md

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/pull/8922.

### CODE REVIEW

File: Teams/deploy-apps-microsoft-teams-landing-page.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ The different types of apps that your end-users can use in Teams are:
 
 ## Core apps
 
-Some default functionality such as activity feed, Teams channels, chat, calendar, and calls are available and pinned by default for ease of access for end-users. As an admin you can modify the default behavior using [setup policy](/microsoftteams/teams-app-setup-policies).
+Some functionality such as activity feed, chat, Teams, assignments (Education tenants), calendar, calls, and files are available and pinned by default for ease of access for end-users. For firstline workers, only activity, shifts, chat, and calling. As an admin you can modify the default behavior using [setup policy](/microsoftteams/teams-app-setup-policies).
```
Comment (JohanFreelancer9) [inline Teams/deploy-apps-microsoft-teams-landing-page.md:46]:
```suggestion
Some functionality such as activity feed, chat, Teams, assignments (Education tenants), calendar, calls, and files are available and pinned by default for ease of access for end-users. For firstline workers, only activity, shifts, chat, and calling are available and pinned. As an admin you can modify the default behavior using [setup policy](/microsoftteams/teams-app-setup-policies).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @dariomws, just a suggestion to make it a complete sentence.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@ashishguptaiitb PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by scanum** [comment]
@ashishguptaiitb PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by ashishguptaiitb** [comment]
Hi @dariomws and @scanum , the PR has conflicts. Also, I don't see a merge option so it may be a permissions issue.

**Comment by dariomws** [comment]
Hi @ashishguptaiitb,

Yes, the PR has conflicts and cannot be merged because you copied and pasted the update into another PR:
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/commit/79e92c75bfdc4696eacd3023fd88f8a733910bc4
Please avoid these practices.

I'll close this one.

**Pull request closed without merging**